backend/Tasque.BLL/MappingProfiles/UserProfile.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoMapper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tasque.Core.Common.DTO;
+using Tasque.Core.Common.Entities;
+
+namespace Tasque.Core.BLL.MappingProfiles
+{
+    public class UserProfile : Profile
+    {
+        public UserProfile()
+        {
+            CreateMap<UserRegisterDto, User>();
+            CreateMap<User, UserDto>();
```

Comment:
consider using ReverseMap() for straight entity -> entityDTO maps

Reply:
Done